Travis fails due to an error which seems unrelated in ddmrp (and random):

```
2020-07-15 08:12:57,085 5735 INFO openerp_test odoo.addons.ddmrp.tests.test_ddmrp: ======================================================================
2020-07-15 08:12:57,085 5735 ERROR openerp_test odoo.addons.ddmrp.tests.test_ddmrp: FAIL: TestDdmrp.test_06_adu_calculation_future_120_days_actual
Traceback (most recent call last):
File "/home/travis/build/OCA/ddmrp/ddmrp/tests/test_ddmrp.py", line 252, in test_06_adu_calculation_future_120_days_actual
self.assertEqual(self.buffer_a.adu, to_assert_value)
AssertionError: 0.0 != 1.0
```

--- a/stock_buffer_route/models/stock_buffer.py
+++ b/stock_buffer_route/models/stock_buffer.py
@@ -38,8 +38,14 @@ class StockBuffer(models.Model):
             lambda route: any(
                 p.location_id in parents
                 for p in route.rule_ids.filtered(
-                    lambda rule: rule.action in ("pull", "pull_push")
+                    lambda rule: rule.action in ("pull", "pull_push", "manufacture")
                 ).mapped("location_src_id")
+            )
+            or any(rule.action == "buy" for rule in route.rule_ids)
+            or any(
+                rule.action == "manufacture"
+                for rule in route.rule_ids
+                if not rule.location_src_id
             )
         )
 

